internal/providers/anthropic.go

```
@@ -0,0 +1,99 @@
+	if err != nil {
+		return Response{}, err
+	}
+	httpReq.Header.Set("Content-Type", "application/json")
+	httpReq.Header.Set("x-api-key", r.apiKey)
+	httpReq.Header.Set("anthropic-version", "2023-06-01")
+
+	resp, err := r.client.Do(httpReq)
+	if err != nil {
+		return Response{}, fmt.Errorf("anthropic: %w", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)